osf/migrations/0085_populate_notification_subscriptions.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from django.db import migrations
+from django.apps import apps
+
+from api.preprint_providers.permissions import GroupHelper
+
+logger = logging.getLogger(__file__)
+
+NotificationSubscription = apps.get_model('osf', 'NotificationSubscription')
+PreprintProvider = apps.get_model('osf', 'PreprintProvider')
+
+def populate_provider_notification_subscriptions(*args):
+    for provider in PreprintProvider.objects.all():
+        helper = GroupHelper(provider)
+        provider_admins = helper.get_group('admin').user_set.all()
+        provider_moderators = helper.get_group('moderator').user_set.all()
+        instance, created = NotificationSubscription.objects.get_or_create(_id='{provider_id}_preprints_added'.format(provider_id=provider._id),
+                                                                           event_name='preprints_added',
```

Comment:
1) The `event_name` should probably be put into `website/notifications/constants`, for consistency with other `NotificationSubscription` events

2) Minor: this could be interpreted to mean multiple things. Consider a more descriptive name, as this is meant to notify moderators that there's been a submission that they should review.